packages/babel-core/src/config/full.js

```diff
@@ -70,21 +74,54 @@ export default function loadFullConfig(
       },
       pass: Array<Plugin>,
     ) {
-      const plugins = config.plugins.reduce((acc, descriptor) => {
-        if (descriptor.options !== false) {
-          acc.push(loadPluginDescriptor(descriptor, context));
-        }
-        return acc;
-      }, []);
-      const presets = config.presets.reduce((acc, descriptor) => {
-        if (descriptor.options !== false) {
-          acc.push({
-            preset: loadPresetDescriptor(descriptor, context),
-            pass: descriptor.ownPass ? [] : pass,
-          });
-        }
-        return acc;
-      }, []);
+      const plugins = config.plugins.reduce(
+        (acc: Plugin[], descriptor: UnloadedDescriptor, i) => {
+          if (descriptor.options !== false) {
+            try {
+              acc.push(loadPluginDescriptor(descriptor, context));
+            } catch (e) {
+              // print special message for `plugins: ["@babel/foo", { foo: "option" }]`
+              if (i > 0 && e.code === "BABEL_UNKNOWN_PLUGIN_PROPERTY") {
```

Comment:
If we are not checking if `config.plugins` is 2, we should also have a test with more plugins.

Reply:
Can do.